docs/core/extensions/snippets/workers/windows-service/Program.cs

```diff
@@ -2,25 +2,21 @@
 using Microsoft.Extensions.Logging.Configuration;
 using Microsoft.Extensions.Logging.EventLog;
 
-using IHost host = Host.CreateDefaultBuilder(args)
-    .UseWindowsService(options =>
-    {
-        options.ServiceName = ".NET Joke Service";
-    })
-    .ConfigureServices(services =>
-    {
-        LoggerProviderOptions.RegisterProviderOptions<
-            EventLogSettings, EventLogLoggerProvider>(services);
+HostApplicationBuilder builder = Host.CreateApplicationBuilder(args);
+builder.Services.AddWindowsService(options =>
+{
+    options.ServiceName = ".NET Joke Service";
+});
```

Comment:
I don't think the brackets are necessary here:

```suggestion
builder.Services.AddWindowsService(options => options.ServiceName = ".NET Joke Service";);
```

Reply:
I think for readability, I'm going to leave this as is. Also, there is specific line highlighting that is in the markdown file too that would also have to be updated.